floris/parallel_floris_model.py

```diff
@@ -89,6 +88,11 @@ def __init__(
         print_timings (bool): Print the computation time to the console. Defaults to False.
         """
 
+        self.logger.warning((
```

Comment:
We could add a link here to docs, but downside is the link won't work until we merge to develop

Reply:
Yeah, as you point out in your comment further down, I think that will be too brittle---hopefully the suggestion will be enough and users will look at the examples/docs to find the `ParFlorisModel`